Portfolio mobile version setup

Mikelobi opened the pull request:
**Portfolio mobile version setup**

In this project:
- **I created the index page.**
- **I added quality style to the project.**
- **I created linter to check errors in the project for standard product.**
- **I created gitignore file to ignore files that are not needed in the project.**
- **I added a Readme file for further details.**

Amrhub commented on index.html:
```diff
@@ -0,0 +1,33 @@
+        <a href="index.html" class="logo">My Logo</a>
+        <nav><img src="image/Union.png" alt="Menu icon"></nav>
+      </header>
+      <div class="container">
```
- [ ] Let's make sure this is a `section` tag better HTML semantic section tag should be the parent of div tag, not the other way around.

Mikelobi replied:
I have updated it

Amrhub commented on index.html:
```diff
@@ -0,0 +1,33 @@
+      </header>
+      <div class="container">
+        <h1 class="detail one">I'm Mariane<br>Glad to see<br>you</h1>
+        <section class="detail two">
```
- [ ] Let's make sure our introduction paragraph is a `p` tag for better HTML semantic

Mikelobi replied:
I have made changes to it

Amrhub reviewed (changes requested):
Hi @Mikelobi,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

## To Highlight ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Highly organized and clean code ✔️ 
## Required Changes ♻️
_Check the comments under the review._
![](https://media.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Mikelobi committed:
Add github workflow

I added linter to check errors in HTML, CSS and Web contents.

Mikelobi committed:
Create .gitignore file

This will help in removing unwanted files from the project

Mikelobi committed:
Add images and logos

I add images and logos for the project

Mikelobi committed:
Add index page

Mikelobi committed:
Add readme file

I added useful information about the project for further references

gemmen29 reviewed (changes requested):
Hi @Mikelobi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
- Please make the changes the previous reviewer asked your code is still the same 
![image](https://user-images.githubusercontent.com/24703171/154254602-c470d2a0-2b47-40bb-9d4d-0de25251ef77.png)

this is the code that I found in the branch `Portfolio-Mobile-Version-Setup`. 
- Kindly change the `div` that takes class `container` to be `section` instead of `div` for sementic.
- Please change the `section` that takes classes ` detail ` , `two` to be `p` instead of `section` for semantic

![](https://media.giphy.com/media/tKyrc0NTlEcGWdThp4/giphy.gif)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Mikelobi committed:
Add css file

- I added style to the project to make it readable and easily understandable.

- It will also help the navigation through the page easier.

gemmen29 reviewed (approved):
Hi @Mikelobi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![](https://media.giphy.com/media/mCIjCgs3nWQWfJZvPA/giphy-downsized-large.gif)

## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Mikelobi committed:
Update index.html

Mikelobi committed:
Updated semantic tags

I updated the semantic tags for a standard coding practice.
